### ISSUE TITLE
Resnet152-yolo very low accuracy

### ISSUE THREAD

**Issue by aNothing** [AlexeyAB/darknet#1076]
I use this command to train 3000 images for 3 classes,
**./darknet detector train data/obj.data  cfg/resnet152_yolo.cfg resnet152.201 -gpus 0,1**

The cfg file is like this:
[net]
batch=64
subdivisions=64
height=416
width=416
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1

learning_rate=0.00001
max_batches = 45000
policy=steps
steps=100,25000,35000
scales=10,.1,.1

The following are the same as the resnet152_yolo.cfg except classes=3, filter=40.
After 6000 iterations, I use  darknet detector map to get the mAP.

![image](https://user-images.githubusercontent.com/18108332/41694046-e0190148-753a-11e8-9879-ca5d0804dec4.png)

The accuracy is very low. Below is my compiling file for darknet.
**GPU=1
CUDNN=1
CUDNN_HALF=1
OPENCV=1
AVX=0
OPENMP=0
LIBSO=0**

My system is Ubuntu 16.04, CUDA 9.0, cuDNN 7.0.5.
Thanks for any suggestions!

**Comment by AlexeyAB** [comment]
`resnet152_yolo.cfg` - is just for experiments, it doesn't give high accuracy for most of datasets.

* What GPU do you use?
* Can you train any other yolo model successfully on the same dataset?
* Did you use `stopbackward=1` in cfg?
* Try to move `stopbackward=1` to earlier layer.

**Comment by aNothing** [comment]
Hi, AlexeyAB. Thanks for your reply.

- GPU: GEFORCE GTX 1080 * 2

- Yes, I can train yolov3.cfg

- I did not use stopbackward=1 in cfg

**Comment by AlexeyAB** [comment]
@aNothing 

* try to un-comment this line: https://github.com/AlexeyAB/darknet/blob/1e9d1fcedf1a361bcdb384f15b5b14bdb526576d/build/darknet/x64/resnet152_yolo.cfg#L1434

* or try to move `stopbackward=1` there: https://github.com/AlexeyAB/darknet/blob/1e9d1fcedf1a361bcdb384f15b5b14bdb526576d/build/darknet/x64/resnet152_yolo.cfg#L1406

**Comment by aNothing** [comment]
@AlexeyAB , I tried to un-comment stopbackward=1 and the accuracy are all 0 after 12k iterations. From 100 to 12k iterations, the mAP value all are 0. In my last training, I commented stopbackward=1 and the result was also bad(<10%).
